src/pages/CreateGarden/index.tsx

```diff
@@ -82,7 +83,7 @@ export const CreateGarden = () => {
       const newGarden: Garden = {
         name: name,
         description: desc,
-        userFireBaseId: userData?.id || "",
+        userFireBaseId: (userData.isLoggedIn && userData.id) || "",
```

Comment:
Yeah we need to refactor this and other components logic,

> disable the create garden button if the user isn't logged in 

we can do it but this page can not be accessed if you are not logged in, so maybe no need to disable

> will we have to do it regardless?

For now we have to do because how I define the type, the reason I did this is because if you are not logged in this state shouldn't exist

Reply:
Understood!